src/pages/editor/EditorCurators.tsx

```diff
@@ -69,9 +70,10 @@ export const EditorCurators = () => {
           <TableHead>
             <TableRow>
               <TableCell>{t('common.name')}</TableCell>
-              <TableCell sx={{ minWidth: { xs: '15rem', md: '40%' } }}>
+              <TableCell sx={{ minWidth: { xs: '30%', md: '50%' } }}>
```

Comment:
tror denne skaper mer rot enn orden atm
```suggestion
              <TableCell>
```